multisrc/overrides/madara/reaperscans/src/ReaperScansFactory.kt

```diff
@@ -1,59 +0,0 @@
-package eu.kanade.tachiyomi.extension.all.reaperscans
-
-import eu.kanade.tachiyomi.multisrc.madara.Madara
-import eu.kanade.tachiyomi.source.SourceFactory
-import eu.kanade.tachiyomi.source.model.SChapter
-import okhttp3.HttpUrl.Companion.toHttpUrl
-import org.jsoup.nodes.Element
-import java.text.SimpleDateFormat
-import java.util.Locale
-
-class ReaperScansFactory : SourceFactory {
-    override fun createSources() = listOf(
-        ReaperScansTr(),
-        ReaperScansFr(),
-    )
-}
-
-abstract class ReaperScans(
-    override val baseUrl: String,
-    lang: String,
-    dateFormat: SimpleDateFormat = SimpleDateFormat("MMMM dd, yyyy", Locale.US),
-) : Madara("Reaper Scans", baseUrl, lang, dateFormat) {
-
-    override fun chapterFromElement(element: Element): SChapter = SChapter.create().apply {
-        val urlElement = element.selectFirst(chapterUrlSelector)!!
-
-        name = urlElement.selectFirst("p.chapter-manhwa-title")?.text()
-            ?: urlElement.ownText()
-        date_upload = urlElement.selectFirst("span.chapter-release-date > i")?.text()
-            .let { parseChapterDate(it) }
-
-        val fixedUrl = urlElement.attr("abs:href").toHttpUrl().newBuilder()
-            .removeAllQueryParameters("style")
-            .addQueryParameter("style", "list")
-            .toString()
-
-        setUrlWithoutDomain(fixedUrl)
-    }
-}
-
-class ReaperScansTr : ReaperScans(
-    "https://reaperscanstr.com",
-    "tr",
-    SimpleDateFormat("MMMMM dd, yyyy", Locale("tr")),
-) {
-
-    // Tags are useless as they are just SEO keywords.
-    override val mangaDetailsSelectorTag = ""
-}
-
-class ReaperScansFr : ReaperScans(
-    "https://reaperscans.fr",
-    "fr",
-    SimpleDateFormat("dd/MM/yyyy", Locale.US),
-) {
-
-    // Migrated from WpMangaReader to Madara.
-    override val versionId = 2
-}
```

Comment:
ReaperScans French is definitely not down. It's only inaccessible from outside France.

![image](https://github.com/keiyoushi/extensions-source/assets/92439990/9980c028-6fd9-4fea-adf7-4822a1873858)

Reply:
> ReaperScans French is definitely not down. It's only inaccessible from outside France.
> 
> ![image](https://private-user-images.githubusercontent.com/92439990/298637551-9980c028-6fd9-4fea-adf7-4822a1873858.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5Mzk5MDEsIm5iZiI6MTcwNTkzOTYwMSwicGF0aCI6Ii85MjQzOTk5MC8yOTg2Mzc1NTEtOTk4MGMwMjgtNmZkOS00ZmVhLWFkZjctNDgyMmExODczODU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIyVDE2MDY0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ5NThmNDk1ZTc1MzVjNjhkYzMxOTQyOTRmMzI1NjBjZjQ0MzYzODA4NTIyYzJlM2UwY2NhM2VkYmQwNjRiYWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.C4Zd5M7oIoE2jf_m68lxYsFAkykKMUCoECAnZpKGdUI)

Wow, Because where I live, the site never loaded, if it's still working I'll fix it.